### DISCUSSION TITLE
Client blocked after several requests

### DISCUSSION THREAD

**Discussion by MathiasDrapier** [miguelgrinberg/Flask-SocketIO#1832] [Q&A]
Hello,

I have a flask and socketio app which acts as a SocketIO server and also a flask API. The SocketIO part works perfectly.
But concerning flask API I have an issue, when a client sends several requests very close in time, it is blocked and never receives a response. This doesn't entirely block the app because my API responds to other clients. 
I got around the problem by adding a timeout in Nginx but this solution doesn't suit me.

Do you have any idea what is going on and what is the solution?

**Comment by miguelgrinberg** [comment]
If the Socket.IO portion works fine, how is Socket.IO influencing this problem? This library doesn't have any logic to block or time limit clients.
